.github/workflows/stage-deploy.yml

```diff
@@ -0,0 +1,28 @@
+name: Deployment
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  deployment:
+    runs-on: ubuntu-latest
+    environment:
+      name: test
+      url: https://stage.codelabs.site
```

Comment:
What is this for?

Reply:
This URL will be displayed at deploy result